lib/tasks/status.rake

```diff
@@ -9,21 +9,26 @@ task status: :environment do
   servers_info = []
   Server.all.each do |server|
     begin
-      response = get_post_req(encode_bbb_uri('getMeetings', server.url, server.secret))
-      meetings = response.xpath('/response/meetings/meeting')
-
+      # Empty defaults
+      meetings = []
       server_users = 0
       video_streams = 0
       users_in_largest_meeting = 0
 
-      meetings.each do |meeting|
-        count = meeting.at_xpath('participantCount')
-        users = count.present? ? count.text.to_i : 0
-        server_users += users
-        users_in_largest_meeting = users if users > users_in_largest_meeting
+      # Scan only enabled servers that are online
+      if server.online && server.enabled
```

Comment:
+1, but it would be nice if this continues fetching the status of online disabled servers:

```suggestion
      # Scan only online servers
      if server.online
```

For maintenance, we disable the server and then keep checking the status before proceeding until it says no meetings so that we don't disrupt any meetings. If only enabled servers are fetched, it makes it harder to determine if any disabled servers are currently in use.

Reply:
Yes, that is right, I did miss that!

Reply:
Yeah I would agree here. If you can change this to only check online servers, this can be merged